yt_dlp/extractor/espn.py

```diff
@@ -279,3 +278,133 @@ def _real_extract(self, url):
             'formats': formats,
             'subtitles': subtitles,
         }
+
+
+class WatchESPNIE(AdobePassIE):
+    _VALID_URL = r'https://www.espn.com/watch/player/_/id/(?P<id>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})'
+    _TESTS = [{
+        'url': 'https://www.espn.com/watch/player/_/id/6eee6cb0-1795-49e3-84ea-766b517e0309',
+        'info_dict': {
+            'id': '6eee6cb0-1795-49e3-84ea-766b517e0309',
+            'ext': 'mp4',
+            'title': 'Miami vs. #2 Duke',
+            'thumbnail': 'https://artwork.api.espn.com/artwork/collections/media/6eee6cb0-1795-49e3-84ea-766b517e0309/default?width=640&apikey=1ngjw23osgcis1i1vbj96lmfqs',
+        },
+        'params': {
+            'skip_download': True,
+        },
+    }, {
+        'url': 'https://www.espn.com/watch/player/_/id/bd1f3d12-0654-47d9-852e-71b85ea695c7',
+        'info_dict': {
+            'id': 'bd1f3d12-0654-47d9-852e-71b85ea695c7',
+            'ext': 'mp4',
+            'title': 'Tue, 1/11 - ESPN FC',
+            'thumbnail': 'https://s.secure.espncdn.com/stitcher/artwork/collections/media/bd1f3d12-0654-47d9-852e-71b85ea695c7/16x9.jpg?timestamp=202201112217&showBadge=true&cb=12&package=ESPN_PLUS',
+        },
+        'params': {
+            'skip_download': True,
+        },
+    }]
+
+    _API_KEY = 'ZXNwbiZicm93c2VyJjEuMC4w.ptUt7QxsteaRruuPmGZFaJByOoqKvDP2a5YkInHrc7c'
+
+    def _call_bamgrid_api(self, path, video_id, data=None, headers={}):
+        if 'authorization' not in headers:
+            headers['authorization'] = 'Bearer ' + self._API_KEY
```

Comment:
not `Authorization`?

Reply:
I don't think capitalization ever matters at all from the server's perspective but if the convention elsewhere in the codebase is to capitalize then I can switch it